Documentation/Signoffs/Raymond Mule-Signoff-Wireless Communication Subsystem(Updated).md

```diff
@@ -58,6 +59,8 @@ The RAK811 transceiver has a very high sensitivity of -148 dBm, enabling it to t
 
 6. The RAK811 Breakout Board will be sending data to and from the computing subsystem to the user interface subsystem. The size of data that will be sent from the computing subsystem to the user interface subsystem and back will approximately be around 30 bits. The maximum payload size that the RAK811 transceiver can send in one data packet is 52 bytes which is signifacntly greater than the needed amount. 
 
+7. The RAK811 Breakout Board must be connected to the computing subsystem in order to transmit data from the computing subsystem to the user interface subsystem. In order connect the RAK811 Breakout Board and the computing subsystem, a RAKDAP1 is used to connect the transceiver to the computing subsystem. The RAKDAP1 is ewuipped with a USB that enables it to connect to any device that possess a USB port, which the computing subsystem has. Before you are able to do this however, you must connect the RAKDAP1 with the RAK811 attached to it to a device that can access the internet. From there you must register and configure the device to how the user wants it using a program given by the manufacturer. Once the setup is complete, the RAKDAP1 with the RAK811 can be connected to the computing subsystem and be operated using commands found in a python library which the computing system will be utilizing.
```

Comment:
This doesn't really address compatibility.

Reply:
Prove that it will work with what it needs to work with the computer you intend to plug it to.